build/cluster-operator-ansible/playbooks/cluster-operator/aws/install_masters.yml

```diff
@@ -0,0 +1,49 @@
+---
+# call openshift-ansible's master provisioning
+- import_playbook: ../../aws/openshift-cluster/install.yml
+
+#- name: setup master ansible hosts
+#  hosts: localhost
+#  gather_facts: false
+#  tasks:
+#  - name: setup the master node group
+#    import_role:
+#      name: openshift_aws
+#      tasks_from: setup_master_group.yml
+
+- name: save kubeconfig into secret
+  hosts: masters[0]
+  gather_facts: false
+  remote_user: root
+  tasks:
+  - name: slurp kubeconfig
+    slurp:
+      src: /root/.kube/config
```

Comment:
I'd prefer /etc/origin/master/admin.kubeconfig

Reply:
changed

Reply:
i run ocp with cluster operator (just to keep us honest :) so /etc/origin/master/admin.kubeconfig does work